DecipherNow/gm-fabric-dashboard
# Fix: Fabric sort by toggles asc/desc

Resolves #1023 

## Proposed Changes

  - Added `ascending` bool to FabricGrid state
  - Added a function to sort service statuses in the correct order: Down, Warning, Stable

## How to Test 

- **Fabric Main View => Group By: Owner/Capability/None (Card and List Views)** 
Check that Sort By Name works. Sort By Status should render in the order Down, Warning, Stable. In both case, selecting the same sort by option twice should toggle asc/desc.
